adaptive_navigation/example/pubspec.yaml

```diff
@@ -6,14 +6,15 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
   adaptive_navigation:
     path: ../
-  adaptive_breakpoints: ^0.0.2
+  adaptive_breakpoints:
+    path: ../../adaptive_breakpoints/
```

Comment:
TODO remove both paths

Reply:
Added TODO for steps after NNBD becomes beta.